config/example-extras.cfg

```diff
@@ -1094,3 +1094,13 @@
 # default_prefix: echo:
 #    Directly sets the default prefix. If present, this value will override
 #    the "default_type".
+
+
+# Filament run out sensor (one may define any number of sections with a
+# "filament_runout_sensor" prefix). When trigered, a message will be sent
+# back and the pause command will be executed in octoprint.
+# By default pin HIGH=FILAMENT PRESENT
+# You can check the state of the sensor using the command
+# QUERY_FILAMENT_SENSOR SENSOR=sensor_name
+#[filament_runout_sensor sensor_name]
+#pin = ^ar4
```

Comment:
The safer configuration here would be `pin = ^!ar4`, with the runout switch connected normally-open between ground and the ar4 pin. That way if there is any wiring fault the sensor will read that there is no filament present.

Reply:
You're correct, but this is in a example file, it should not have the pin parameter predetermined. I think that the correction I must do its to remove the `^ar4` from this line.